cypress/e2e/column-text-link.cy.js

```diff
@@ -0,0 +1,79 @@
+let localUser
+
+describe('Test column text-link', () => {
+
+	before(function() {
+		cy.createRandomUser().then(user => {
+			localUser = user
+		})
+	})
+
+	beforeEach(function() {
+		cy.login(localUser)
+		cy.visit('apps/tables')
+	})
+
+	it('Create', () => {
+		cy.contains('.app-menu-entry--label', 'Tables').click()
+		cy.contains('button', 'Create new table').click()
+		cy.get('.tile').contains('Custom table').click({ force: true })
+		cy.get('.modal__content input[type="text"]').clear().type('Test text-link')
```

Comment:
It might be nice to move common selectors or logic (e.g. to create a table into helper functions)